### ISSUE TITLE
throw error when editing rust file

### ISSUE THREAD

**Issue by alan-w-255** [p00f/nvim-ts-rainbow#3]
when i enable this plugin, then edit the rust file. nvim throw error:

`Error executing lua callback: /usr/local/share/nvim/runtime/lua/vim/treesitter/query.lua:272: invalid start`

I am using:
NVIM v0.5.0-888-g8b2887bd5
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3

**Comment by p00f** [comment]
yeah there's something wrogn with css too

**Comment by p00f** [comment]
can you share the file you're editing and reproduction steps? (like what exactly you did and the line and col numbers of that position)

**Comment by alan-w-255** [comment]
```rust
fn main() {
    let a_rename="hi";
    println!("{}", a_rename);
}

#[test]
fn it_work() {
    assert!(1+1 == 2);
}
```

I put the cursor on the `println`, then press `o` to start a newline, then nvim throw the error.

**Comment by p00f** [comment]
In https://github.com/nvim-treesitter/nvim-treesitter/issues/528 vigoux said this is an upstream issue.

Can you confirm this is the case here too @vigoux?

**Comment by p00f** [comment]
point to be noted is that this doesn't happen when the rainbow module is disabled

**Comment by alan-w-255** [comment]
Yes, it won't happen when i disable 'rust' filetype in rainbow config.

It seem the following code will cause the errors:

```rust
#[test]
fn it_work() {
    assert!(1+1 == 2);
}
```
when i delete those code, there is no more error. but there is no rainbow brackets neither

**Comment by p00f** [comment]
my `~/.local/share/nvim/log` says something like
> RPC: ensure that it is properly synced

but i deleted my log file by mistake

can you show what it says @alan-w-255 ?

**Comment by alan-w-255** [comment]
[log.txt](https://github.com/p00f/nvim-ts-rainbow/files/5613834/log.txt)
@p00f

**Comment by p00f** [comment]
can you try the error-handling branch (with some more editing/testing)?

**Comment by alan-w-255** [comment]
I have test the `error-handing branch`,  there is no more errors. It is fixed.

**Comment by p00f** [comment]
I'll merge the branch soon then.

I'll keep this issue open as a reminder to keep checking if the underlying problem is fixed

**Comment by p00f** [comment]
I've merged the branch

**Comment by p00f** [comment]
I just closed #1 because it seems fixed, maybe it is indeed fixed upstream - @vigoux ?

**Comment by p00f** [comment]
Closing because this is not reproducible anymore
